PresnyaHedonist/Controllers/InfoVC.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  InfoVC.swift
+//  PresnyaHedonist
+//
+//  Created by Alexandr L. on 7/13/21.
+//
+
+import UIKit
+import SafariServices
+import CoreData
+import CallKit
+
+class InfoVC: UIViewController {
+    
+    var place: Place?
+    
+    let context = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
```

Comment:
Nit: context is not actually used.

Reply:
for now) It's a part of adding place to FavoritesVC - but not done yet :)